oauth2_http/javatests/com/google/auth/oauth2/MockMetadataServerTransport.java

```diff
@@ -300,9 +300,7 @@ public LowLevelHttpResponse execute() throws IOException {
         GenericJson content = new GenericJson();
         content.setFactory(OAuth2Utils.JSON_FACTORY);
         if (requestStatusCode == 200) {
-          for (Map.Entry<String, String> entrySet : s2aContentMap.entrySet()) {
-            content.put(entrySet.getKey(), entrySet.getValue());
-          }
+          content.put("s2a", s2aContentMap);
```

Comment:
nit: Can you use the Env Var from above?

Reply:
Done in the latest commit.